OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -52,147 +52,601 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        Wax stains are visible on <locus target="#31v #72v"/>. The parchment is stained and crumpled on <locus target="#74"/>.
+                                        Tears repaired on <locus target="#2 #35 #100 #106 #118"/>.
+                                    </condition>
+                                </supportDesc>
+                                
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="24">
+                                        <locus from="1ra" to="123vb"/>
+                                        <note corresp="#textarea #margin">Data on text area and margin dimensions taken from <locus target="#45r"/>.</note>
+                                        <!--      <dimensions unit="mm" xml:id="textarea1">
+                                        <height></height>
+                                        <width></width>
+                                       <!- - no text area provided: added to REQUEST FOR FURTHER ACTION list- ->
+                                    </dimensions>-->
+                                        <dimensions type="margin" unit="mm" xml:id="margin1">
+                                            <dim type="top">31</dim>
+                                            <dim type="bottom">56</dim>
+                                            <dim type="right">47</dim>
+                                            <dim type="left">15</dim>
+                                            <dim type="intercolumn">15</dim>
+                                        </dimensions>
+                                        <note> The characters per line are 14-17.</note>
+                                        <ab type="punctuation" subtype="Dividers">
+                                            Several caesura signs are attested in the manuscript:
+                                            the <foreign xml:lang="la">crux ansata</foreign> is written in the left margin on <locus target="#65va #74vb"/>;
+                                            the <foreign xml:lang="la">coronis</foreign> sign is written in the left margin, e.g. on <locus target="#3vb #56vb #101va #116va"/>, etc.;
+                                            the <foreign xml:lang="la">crux ansata</foreign> and the <foreign xml:lang="la">coronis</foreign> are written together on <locus target="#90va #93rb #97va #98va #98vb"/>.
+                                            The word <foreign xml:lang="gez">ቁም፡</foreign> is written at the end of the line or in the right margin, most often in red ink, e.g. on <locus target="#5rb #59vb #97ra #116rb"/>, etc.
+                                            Chains of black and red right-pointing chevrons in the left margin, indicating quotations from the Old Testament embedded in the Gospel text, are found e.g. on <locus target="#3rb #14ra #18rb #44rb #93vb"/>, etc.
```

Comment:
I am not sure about "chevron" maybe "v-shaped" or "right pointing arrow-shaped" marks? ...etc. I would either add "are written in the left margin to indicate quotations" here or deleted "is written" in the previous list (I think I would delete "is written for sake of concision)

Reply:
"Chevron" was used in a Working Rule document which circulated few years ago among the Ethio-SPaRe team and I retrieved that term to describe those chains of marks. I'm fine with "v-shaped marks"

Reply:
Again, a bit like with "caesura marker" I am thinking also about what an informed English reader would understand, but if it has been used, let's keep chevron, but let's maybe say "chevron signs"